docs/usage/INVENTORY_SYNC.md

```diff
@@ -100,8 +100,8 @@ Having an inventory entry, with a missing supply channel reference, could be pro
 - `batchSize`
 a number that could be used to set the batch size with which inventory entries are fetched and processed with,
 as inventory entries are obtained from the target CTP project in batches for better performance. The algorithm accumulates up to
-`batchSize` inventory entries from the input list, then fetches the corresponding inventory entries from the target CTP project
-in a single request, and then performs the update actions needed. Playing with this option can slightly improve or reduce processing speed.
+`batchSize` inventory entries from the input list, then fetch the corresponding inventory entries from the target CTP project
```

Comment:
Why? It should be `fetches`, since it the algorithm which does the _fetch_ ing is singular.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/319/commits/1a9877eabd76eddffe0b501bb21204936c0a815b